src/routes/(app)/namespaces/[namespace]/workflows/[workflow]/[run]/history.json/+page.ts

```diff
@@ -0,0 +1,15 @@
+import { redirect } from '@sveltejs/kit';
+
+import type { PageLoad } from './$types';
+
+import { getEndpointForRawHistory } from '$lib/services/events-service';
+
+export const load: PageLoad = async function ({ params }) {
```

Comment:
Just thinking through this auth flow, the child page servers get called first, so we might need ({ params, parent }) {

await parent(); => This would await top-level layout.ts and trig the auth redirect